### PR TITLE
submission homework week2

### PR DESCRIPTION
**Pull request by Mehmet-Ergorgec**

### CODE REVIEW

File: homework-submission/week2/src/App.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import React, { useState } from 'react';
+import './App.css';
+import uuid from 'uuid';
+import List from './components/list';
+import ListHeader from './components/listHeader';
+import Input from './components/Input';
+import Button from './components/Button';
+
+function App() {
+  const [Todos, setTodos] = useState([
```
Comment (ArcoMul) [inline homework-submission/week2/src/App.js]: In Javascript we don't write variable name with a capital letter (`Todos`), only classes are written with a capital

File: homework-submission/week2/src/components/Button/index.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function Button({ type = 'submit', value }) {
+  return (
+    <div>
```
Comment (ArcoMul) [inline homework-submission/week2/src/components/Button/index.jsx]: The encapsulating `<div>` is not needed

File: homework-submission/week2/src/components/Input/index.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+function Input({ type = 'text', placeholder, value, name, handleChangeInput }) {
+  return (
+    <div>
```
Comment (ArcoMul) [inline homework-submission/week2/src/components/Input/index.jsx]: Same, `<div>` is not needed

File: homework-submission/week2/src/components/list/index.jsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import '../list/index.css';
+
+function List({ id, description, deadline, handleRemoveItem }) {
```
Comment (ArcoMul) [inline homework-submission/week2/src/components/list/index.jsx]: For a component called List I'd expect it to render multiple list items, your component only renders one list item, try to find a more fitting name :-)

File: homework-submission/week2/src/App.js [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+    }
+  };
+
+  const handleRemoveItem = id => {
+    const removeItem = Todos.filter(todo => todo.id !== id);
+    setTodos(removeItem);
+  };
+
+  const [showTodoLIst, setShowTodoLIst] = useState(false);
+  const [buttonText, setButtonText] = useState(false);
```
Comment (ArcoMul) [inline homework-submission/week2/src/App.js]: These two state properties could be moved to the form state. Also the `buttonText` property has in the end the same functionality as `showTodoList`, you could pick the button text based on the `showTodoList` value.

File: homework-submission/week2/src/App.js [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+    <div className="App-header">
+      <ListHeader listName="Todo List" />
+      <form className="form" onSubmit={handleSubmit}>
+        <Input
+          placeholder="description..."
+          value={form.description}
+          name="description"
+          handleChangeInput={handleChangeInput}
+        />
+        <Input
```
Comment (ArcoMul) [inline homework-submission/week2/src/App.js]: Would be good to use a date input here

File: homework-submission/week2/src/App.js (hunk 5 above)
Comment (Mehmet-Ergorgec) [reply homework-submission/week2/src/App.js]: Thank you. They are doing the same job. I changed and commited my changes

File: homework-submission/week2/src/App.js (hunk 6 above)
Comment (Mehmet-Ergorgec) [reply homework-submission/week2/src/App.js]: I did not know that. Thank you. It is good to use it. I implemented it. But I wanted to show user also placeholder. I did it in onFocus function. But basically can do it only type='date'

**Comment by Mehmet-Ergorgec** [comment]
Although I wrote my remove item function in my first commit, after my second commit changes affected my code. I have not realized it.  Now I fixed them all.

**Pull request closed without merging**